### ISSUE TITLE
Add option to show thumbnails for images in Plone 5  content views

### ISSUE THREAD

**Issue by fgrcon** [plone/Products.CMFPlone#1074]
**This is more or less deprecated and solved by other means see coment down at the bottom**

![image](https://cloud.githubusercontent.com/assets/1640384/10071524/c5e5b7d4-62ba-11e5-8fe2-7283852cc2e0.png)

![image](https://cloud.githubusercontent.com/assets/1640384/10071574/1be2f336-62bb-11e5-9f7d-0b263dd36e56.png)

![rename](https://cloud.githubusercontent.com/assets/1640384/10307294/0093c926-6c2e-11e5-8709-842cd6db4499.jpg)

Affected components: 
* mockup pattern structure
* plone.app.content

requires fix https://github.com/plone/Products.CMFPlone/issues/1129 (plone.app.dexterity, ploneapp.contenttypes)

**Comment by fgrcon** [comment]
@vangheem  @jensens ....:
After cleaning up all templates (as far i could spot hem all) I tried to unify everything as much as possible. Thumbs are shown in all listings whe item is an Image or item has a leadimage. This also applies to folder contents in patt structure as well. So there is no need to extend the vocubalary any more and available Column Thumb is not needed anymore either.
![foldercontents](https://cloud.githubusercontent.com/assets/1640384/10817280/081515b6-7e39-11e5-8854-00066ef97e0e.jpg)

The only important change is in p.a.contenttypes: redefined metadata field: getIcon.
**I suppose this will require an upgrade step to reindex?**

We still have a small problem: we need to replace portal_type.toLowerCase() with a js function that more or less implemets normalizeString (e.g: 'news Item' --> 'news-item' also important for custom types). I tried a lot  but got totally lost in  the js-stuff .
